chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
We don't do this in any case — these should just use `var` on first assignment, or `let` if limited to block scope.

My argument for `var` has been that being able to (re)initialize variables at function scope close to where they're used without worrying about whether they're declared above or below is a good feature.